client/src/components/Page/PageList.vue

```diff
@@ -61,8 +60,9 @@
                 </span>
                 <a
                     v-if="!row.item.published && !row.item.shared && !row.item.importable"
-                    :href="shareLink(row.item)"
-                    class="share-this-page">
+                    href="javascript:void(0)"
```

Comment:
Please avoid void links, as these are semantically incorrect. Having a void href makes this element no longer be a link, and it breaks on non-click actions.

You could either use a button styled like a link here, or you can have the click event prevent default behavior and keep the correct link in the href. With latter non right click actions (e.g. middle click, or copy link) will still preform as expected for links.